src/launcher/java/org/truffleruby/launcher/options/OptionsCatalog.java

```diff
@@ -76,6 +76,18 @@
             "ruby.verbosity", 
             "Verbosity", 
             Verbosity.FALSE);
+    public static final BooleanOptionDescription ARGV_GLOBALS = new BooleanOptionDescription(
+            "ruby.argv_globals", 
+            "Parse options in script argv into global variables.", 
+            false);
+    public static final StringArrayOptionDescription ARGV_GLOBAL_VALUES = new StringArrayOptionDescription(
+            "ruby.argv_global_values", 
+            "Parsed options from script argv with a value.", 
+            new String[]{});
+    public static final StringArrayOptionDescription ARGV_GLOBAL_FLAGS = new StringArrayOptionDescription(
+            "ruby.argv_global_flags", 
+            "Parsed options from script argv acting as flags (no value).", 
+            new String[]{});
```

Comment:
Are the flags not the same as if they had the value `true`?
The problem is then differentiating the boolean true and the String "true" since both have to be passed as String in options?